src/Core/DynamicMetadata/SubjectArea.php

```diff
@@ -88,6 +88,9 @@ public static function getName()
         return 'subject_area';
     }
 
+    /**
+     * @return self
```

Comment:
here we used `@return self` (See above comment)

Reply:
fixing to `$this` to be more precise